tests/ReCaptcha.Tests/Services/RecaptchaServiceTests.cs

```diff
@@ -62,7 +62,7 @@ public void Initialize()
             };
 
             _httpClientFactory = new Mock<IHttpClientFactory>();
-            _httpClientFactory.Setup(instance => instance.CreateClient(It.Is<string>(val => val == RecaptchaServiceConstants.RecaptchaServiceHttpClientName)))
+            _httpClientFactory.Setup(instance => instance.CreateClient(RecaptchaServiceConstants.RecaptchaServiceHttpClientName))
```

Comment:
Is there a reason behind this change in these tests? The same change was made on lines 133 and 167.

Reply:
hm, it's a consequence of first this expression having to be removed to match the interface of the previous handler factory, and then me quickfixing it without checking the context :P. I think Moq is actually going to do the same thing in both versions, but indeed no point having this change. I just didn't look at the complete changeset in the end.